client/my-sites/importer/section-import.jsx

```diff
@@ -261,9 +260,15 @@ class SectionImport extends Component {
 	};
 
 	renderImportersList() {
+		const { translate } = this.props;
+		const isSpecificImporter = ! isEmpty( this.state.importers );
+		const sectionHeaderLabel = isSpecificImporter
+			? translate( 'Importing content from:' )
+			: translate( 'I want to import content from:' );
```

Comment:
Similarly here. Translators will need to know what sort of the thing the object in the sentence is.